packages/stark-core/src/validation/validator.intf.ts

```diff
@@ -1,16 +1,96 @@
 import { Validator } from "class-validator";
 
+/**
+ * Interface that defines validators usable in a Stark Project
+ */
 export interface StarkValidator extends Validator {
+	/**
+	 * Function that allows the developer to set a minimum and/or maximum of selected items.
```

Comment:
"Validates that the size of the given array is between the minimum and maximum limits defined" ;)